src/Resources/views/Base/scripts.html.twig

```diff
@@ -5,6 +5,9 @@
 <script>window.jQuery.ui || document.write('<script src="{{ asset(assetsBasePath ~ '/jquery-ui/jquery-ui.min.js') }}"><\/script>')</script>
 {% endif %}
 
-<script src="{{ asset('bundles/cmftreebrowser/js/cmf_tree_browser.' ~ treeAdapter ~ '.js?' ~ 'now'|date('U')) }}" type="text/javascript"></script>
-
-<link rel="stylesheet" href="{{ asset('bundles/cmftreebrowser/css/cmf_tree_browser.' ~ treeAdapter ~ '.css') }}" media="all"/>
+<script>
+if (jQuery.fn.cmfTree === undefined) {
+    document.write('<script src="{{ asset('bundles/cmftreebrowser/js/cmf_tree_browser.' ~ treeAdapter ~ '.js' ~ (app.debug ? 'now'|date('?U') : '')) }}"><\/script>');
```

Comment:
So we would either load them throght admin-pool (SonataPhpcrSonataAdminBundle) or write them into script section manually (standalone treebrowser-bundle)? Yes i really like that.